tutorials/postgres_forcedsource.ipynb

```diff
@@ -12,7 +12,7 @@
    "metadata": {},
    "source": [
     "Owner: **Joanne Bogart [@jrbogart](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@jrbogart)**  \n",
-    "Last Verified to Run: **2020-08-03**\n",
+    "Last Verified to Run: \n",
```

Comment:
Add date? 
```suggestion
    "Last Verified to Run: 2020-11-24\n",
```

Reply:
Reverted to previous version, which was correct.